crashsimilarity/downloader.py

```diff
@@ -32,6 +33,19 @@ class BugzillaDownloader(Downloader):
     def __init__(self, cache=None):
         super().__init__(cache)
 
+    @staticmethod
+    def _clean_signatures(signatures):
+        clean_signatures = set()
+        for sig in signatures.split('\r\n'):
+            pos = sig.find('[@')
```

Comment:
You could do a single substring if you do something like:
```
start_pos = ...
end_pos = ...
if start_pos != -1 and end_pos != -1:
    ...
```

Reply:
Forget it, the pre-existing code was already doing this, so it's fine. We can change it later.